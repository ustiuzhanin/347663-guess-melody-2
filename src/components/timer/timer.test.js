import React from "react";
import renderer from "react-test-renderer";
import Timer from "./timer.jsx";

test(`Timer's Snapshot`, () => {
  const tree = renderer.create(<Timer minutes={4} seconds={30} />).toJSON();

  expect(tree).toMatchSnapshot();
});
