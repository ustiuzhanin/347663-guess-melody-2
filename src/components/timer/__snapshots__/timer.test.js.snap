// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer's Snapshot 1`] = `
<div
  className="timer__value"
>
  <span
    className="timer__mins"
  >
    4
  </span>
  <span
    className="timer__dots"
  >
    :
  </span>
  <span
    className="timer__secs"
  >
    30
  </span>
</div>
`;
